app/controllers/courses_controller.rb

```diff
@@ -71,6 +71,7 @@ def assign_start_date
         @start_date = start_date(@occurrences)
         @started = (@start_date <= Date.today)
       else # This shouldn't happen but some test data has no occurrences
+        Sentry.capture_message('Attempted to determine start date of a course with no occurrences', extra: { course_template_no:  @course&.course_template_no })
```

Comment:
From further up in the file I don't think this `&.` is achieving anything since a `nil` value of `@course` would error elsewhere. May be an idea to remove it so it doesn't inadvertently hide a future regression.

Reply:
Sounds reasonable. My concern was that the Sentry message would end up with a misleading `nil` value but you're right that the Sentry message erroring or not sending would be a more unhelpful regression. Happy for this to be left as is.